website/src/tech-guide.md

```diff
@@ -0,0 +1,119 @@
+
+* Application Hosting  
+  * Decide where the application will be hosted (e.g. cloud infrastructure, on-premise infrastructure). To ensure that CiviForm prioritizes the needs of the community, let CiviForm Governance know which Cloud, hosting, or on-premises environment that you are deploying into.
+
+* Deployment  
+  * Review the Initial Deployment Guide for technical requirements: [https://docs.civiform.us/it-manual/sre-playbook/initial-deployment](https://docs.civiform.us/it-manual/sre-playbook/initial-deployment)
+
+* Integration with existing systems or leveraging existing data  
+  * Consider which integrations may be required for key functionality such as Esri ArcGIS integration for Eligibility Checking or API integrations for referrals.  
+  * Review the [CiviForm API](https://docs.civiform.us/it-manual/api).
+
+* Authentication  
+  * Review the playbook to help choose an authentication provider: [https://docs.civiform.us/it-manual/sre-playbook/initial-deployment/authentication-providers](https://docs.civiform.us/it-manual/sre-playbook/initial-deployment/authentication-providers)
+
+* Ensure that the infrastructure will be paid for and maintained for the entire length of the project.  (Name individuals or group leads).  
+  * What do each team’s project commitments look like in 6 months, 1 year, 5 years?  
+  * What do each team’s staffing commitments look like in the next 6 months, 1 year, 5 years?
+
+* Integration with Existing Systems  
+  * Compatibility assessment: Evaluate how CiviForm integrates with your current systems (CRM, data hub, referral systems, GIS).  
+  * Data migration: Develop a plan to transition application forms from existing systems to CiviForm.  
+  * API utilization: Explore using CiviForm APIs to connect with other platforms for seamless data exchange.  
+      
+* Optional: Onboard engineers to CiviForm code base as per:  
+  * [https://github.com/civiform/civiform/wiki/Technical-contribution-guide](https://github.com/civiform/civiform/wiki/Technical-contribution-guide)  
+  * [https://github.com/civiform/civiform/wiki/New-Full-Time-SWE-Onboarding-Material](https://github.com/civiform/civiform/wiki/New-Full-Time-SWE-Onboarding-Material)
+
+## Preparation for Launch 
+
+* Application Production and Launch Planning  
+  * Agree on the criteria that must be met before an application is publicly launched (e.g. code testing, Quality Assurance (QA), User Research, etc.)  
+  * Ensure you have a plan for updating to new release versions and are comfortable doing so.  
+  * Become familiar with how to turn on and off feature flags (and other settings) via the Admin Settings Panel or your deployment configuration.  
+  * Plan and test a database recovery process for your deployed instance. For example, [restoring the db to a previous snapshot for AW](https://docs.civiform.us/it-manual/sre-playbook/database-disaster-recovery)S.   
```

Comment:
looks like the S in AWS got split to after the link

Reply:
Fixed in the original. It will fix the next time I do a PR for this doc. Thanks!